stdlib.sh

```diff
@@ -804,6 +804,13 @@ layout_node() {
   PATH_add node_modules/.bin
 }
 
+# Usage: layout opam
+#
+# Sets environment variables from `opam env`.
+layout_opam() {
```

Comment:
```suggestion
layout_opam() {
  export OPAMROOT=$(direnv_layout_dir)/opam
```
Would it make sense to set a per-project opam root maybe?

Reply:
I think that should be `OPAMSWITCH`. If you set it, opam env returns an error if the opam switch is not initialized. In a sense, this is expected behavior.